ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostView.java

```diff
@@ -326,11 +327,22 @@ public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event) {
     if (currentActivity != null && !currentActivity.isFinishing()) {
       mDialog.show();
       if (context instanceof Activity) {
-        mDialog
-            .getWindow()
-            .getDecorView()
-            .setSystemUiVisibility(
-                ((Activity) context).getWindow().getDecorView().getSystemUiVisibility());
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.R) {
```

Comment:
The `StatusBarModule` makes this same if/else check, so reimplementing it here:
https://github.com/facebook/react-native/blob/c4f9556f7e98454e2061ca7e121099ebbbddd4c9/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java#L193-L204